specifications/api/swagger.yaml

```diff
@@ -7379,6 +7379,24 @@ paths:
             schema: *ApiSharedWalletPatchData
       responses: *responsesPatchSharedWallet
 
+  /shared-wallets/{walletId}/transactions-construct:
+    post:
+      operationId: constructSharedTransaction
+      tags: ["Transactions New"]
+      summary: Construct
+      description: |
+        <p align="right">status: <strong>under development</strong></p>
+
+        Create a transaction to be signed from the shared wallet.
```

Comment:
I'd be for adding an additional note here explicitely which elements of construct we are expecting to work with a :warning:. (That's because rendered API doc shows all elements of request body `ApiConstructTransactionData` which might suggest all of them work)

Reply:
done